lib/signs/bus.ex

```diff
@@ -9,6 +9,7 @@ defmodule Signs.Bus do
   @drawbridge_minutes_spanish "152"
   @drawbridge_soon_spanish "157"
   @alert_levels [:station_closure, :suspension_closed_station]
+  @sl_route_ids MapSet.new(["741", "742", "743", "746"])
```

Comment:
Should we include `749` and `751` here for completeness? Or maybe call this `@sl_waterfront_route_ids`.

Reply:
I think a more specific name is better here. `749` and `751` only show on Nubian and those are not configurable on the SL page in SignsUI.